File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT0122U00GrdDcodeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
    public class XRT0122U00GrdDcodeInfo
    {
        private String _buwiBunryu;
        private String _buwiCode;
        private String _buwiName;
        private String _sortSeq;
        private String _key;

        public String BuwiBunryu
        {
            get { return this._buwiBunryu; }
            set { this._buwiBunryu = value; }
        }

        public String BuwiCode
        {
            get { return this._buwiCode; }
            set { this._buwiCode = value; }
        }

        public String BuwiName
        {
            get { return this._buwiName; }
            set { this._buwiName = value; }
        }

        public String SortSeq
        {
            get { return this._sortSeq; }
            set { this._sortSeq = value; }
        }

        public String Key
        {
            get { return this._key; }
            set { this._key = value; }
        }

        public XRT0122U00GrdDcodeInfo() { }

        public XRT0122U00GrdDcodeInfo(String buwiBunryu, String buwiCode, String buwiName, String sortSeq, String key)
        {
            this._buwiBunryu = buwiBunryu;
            this._buwiCode = buwiCode;
            this._buwiName = buwiName;
            this._sortSeq = sortSeq;
            this._key = key;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT0122U00GrdDcodeInfo.java
package nta.med.data.model.ihis.xrts;

import java.io.Serializable;

public class XRT0122U00GrdDcodeInfo implements Serializable {
	private String buwiBunryu;
	private String buwiCode;
	private String buwiName;
	private Double sortSeq;
	private String key;

	public XRT0122U00GrdDcodeInfo(String buwiBunryu, String buwiCode,
			String buwiName, Double sortSeq, String key) {
		super();
		this.buwiBunryu = buwiBunryu;
		this.buwiCode = buwiCode;
		this.buwiName = buwiName;
		this.sortSeq = sortSeq;
		this.key = key;
	}

	public String getBuwiBunryu() {
		return buwiBunryu;
	}

	public void setBuwiBunryu(String buwiBunryu) {
		this.buwiBunryu = buwiBunryu;
	}

	public String getBuwiCode() {
		return buwiCode;
	}

	public void setBuwiCode(String buwiCode) {
		this.buwiCode = buwiCode;
	}

	public String getBuwiName() {
		return buwiName;
	}

	public void setBuwiName(String buwiName) {
		this.buwiName = buwiName;
	}

	public Double getSortSeq() {
		return sortSeq;
	}

	public void setSortSeq(Double sortSeq) {
		this.sortSeq = sortSeq;
	}

	public String getKey() {
		return key;
	}

	public void setKey(String key) {
		this.key = key;
	}

}
